src/templates_front/templator_front.org

```diff
@@ -3927,13 +3957,14 @@ contains
    subroutine trexio_str2strarray(str_flat, max_num_str, max_len_str, str_array)
      implicit none
 
-     integer(8), intent(in), value   :: max_num_str  ! number of elements in strign array
-     integer, intent(in), value      :: max_len_str  ! maximum length of a string in an array
-     character, intent(in)           :: str_flat(*)
-     character(len=*), intent(inout) :: str_array(*)
+     integer(c_int64_t), intent(in), value   :: max_num_str  ! number of elements in strign array
+     integer, intent(in), value              :: max_len_str  ! maximum length of a string in an array
+     character(kind=c_char), intent(in)      :: str_flat(*)
+     character(len=*), intent(inout)         :: str_array(*)
 
      character(len=max_len_str)  :: tmp_str
-     integer(8) :: len_flat, i, j, k, ind
+     integer(c_int64_t) :: i, j, k, ind, offset
```

Comment:
`offset` should be removed and these two lines merged

Reply:
OK